roles/kerberos5/tasks/client.yml

```diff
@@ -0,0 +1,81 @@
+---
```

Comment:
I'm not sure if all the magic below is really needed, or, in fact, correct. But I guess it's just copypaste that 'just works', so it will do for starters.

Nevertheless, I'd be a bit happier if it came from a shared component which would receive some more reviews, love and care eventually. Please add a TODO item or smth like that.

Reply:
The `TODO` files tend to be permanent so there is no reason to write it.

Regarding shared component: at the moment I don't see architecture because components are changing.